example/spec/testAPI/APISuccessSpec.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+const fs = require('fs-extra');
+const { loadConfig } = require('../helpers/testUtils');
+const sleep = require('sleep');
+const {
+  aws: { s3 },
+  stringUtils: { globalReplace },
+  testUtils: { randomStringFromRegex }
+} = require('@cumulus/common');
+const { createGranuleFiles } = require('../helpers/granuleUtils');
+const {
+  buildAndExecuteWorkflow,
+  conceptExists
+} = require('@cumulus/integration-tests');
+const config = loadConfig();
+const taskName = 'IngestGranule';
+const granuleRegex = '^MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006.[\\d]{13}$';
+const testDataGranuleId = 'MOD09GQ.A2016358.h13v04.006.2016360104606';
+const { api: apiTestUtils } = require('@cumulus/integration-tests');
+
+/**
+ * Set up files in the S3 data location for a new granule to use for this
+ * test. Use the input payload to determine which files are needed and
+ * return updated input with the new granule id.
+ *
+ * @param {string} bucket - data bucket
+ * @param {string} granuleId - granule id for the new files
+ * @param {string} inputPayloadJson - input payload as a JSON string
+ * @returns {Promise<Object>} - input payload as a JS object with the updated granule ids
+ */
+async function setupTestGranuleForAPI(bucket, granuleId, inputPayloadJson) {
+  const baseInputPayload = JSON.parse(inputPayloadJson);
+
+  await createGranuleFiles(
+    baseInputPayload.granules[0].files,
+    bucket,
+    testDataGranuleId,
+    granuleId
+  );
+
+  const updatedInputPayloadJson = globalReplace(inputPayloadJson, testDataGranuleId, granuleId);
+
+  return JSON.parse(updatedInputPayloadJson);
+}
+
+/**
+ * Checks for granule in CMR until it get the desired outcome or hits
+ * the number of retries.
+ *
+ * @param {string} CMRLink - url for grnaule in CMR
+ * @param {string} outcome - desired outcome
+ * @param {string} retries - number of remaining tries
+ * @returns {boolean} - whether or not the granule exists
+ */
+async function waitForExist(CMRLink, outcome, retries) {
+  if (retries === 0) {
+    console.log('Out of retries.');
+    return false;
+  }
+  const existsCheck = await conceptExists(CMRLink);
+  if (existsCheck !== outcome) {
+    sleep.sleep(2);
+    console.log('Retrying...');
+    return waitForExist(CMRLink, outcome, (retries - 1));
+  }
+  else {
+    return true;
+  }
+}
+
+describe('The Cumulus API', () => {
+  let workflowExecution = null;
+  const collection = { name: 'MOD09GQ', version: '006' };
+  const provider = { id: 's3_provider' };
+  const inputPayloadFilename = './spec/testAPI/testAPI.input.payload.json';
+  let inputPayload;
+  let granuleId;
+
+  beforeAll(async () => {
```

Comment:
I'm wondering if we should put the api tests under spec/ingestGranule/IngestGranuleSuccessSpec.js.  The setup and workflow are the same.